frontend/src/components/form/hash-input.jsx

```diff
@@ -0,0 +1,59 @@
+import { createEffect, createSignal } from "solid-js";
+import { styled } from "solid-styled-components";
+import icon from "../../assets/copy-ico.svg";
+
+const HashInput = (props) => {
```

Comment:
This input should have its signal that represents value because we want to read data from the HTML `input` on the `Edit` page. But we also should have an ability to change this value from outside